src/Logger-MCC_DAQ/main.py

```diff
@@ -27,37 +27,43 @@
 # Contribution: We like to thank TU Dresden/Shayan Miri for providing the initial version of this driver.
 
 # SweepMe! device class
-# Type: Logger
-# Device: Measurement Computing Corporation DAQ devices
+# *Module: Logger
+# *Device: Measurement Computing Corporation DAQ devices
```

Comment:
This is the header from the Logger template:
# SweepMe! driver
# * Module: Logger
# * Instrument: Template
Maybe it's worth to adhere to those when changing the header anyway